### PR TITLE
skip prompt during installing stackdriver agent

### PR DESCRIPTION
**Pull request by zhanbo72**
https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/issues/297

### CODE REVIEW

**Comment by zhanbo72** [comment]
I signed it!

File: stackdriver/stackdriver.sh [hunk 1]
```diff
@@ -23,4 +23,4 @@
 
 # Download and run the Stackdriver installation script
 curl -sSO https://dl.google.com/cloudagents/install-monitoring-agent.sh
-sudo bash install-monitoring-agent.sh
+echo "Y" | sudo bash install-monitoring-agent.sh
```
Comment (jmikula) [inline stackdriver/stackdriver.sh]: Consider:
yes | sudo bash install-monitoring-agent.sh

**Comment by zhanbo72** [comment]
Replaced the 'echo "Y"' with 'yes'.

**Comment by zhanbo72** [comment]
the stackdriver-agent init action script cannot fixed by simplely passing 'yes' to prompt up.

Actually an easier way to install Stackdriver agent is add property 'dataproc:dataproc.monitoring.stackdriver.enable=true' in the create cluster command, for exmaple: "gcloud dataproc clusters create test-cluster --properties 'dataproc:dataproc.monitoring.stackdriver.enable=true'"

Close this pull request.

**Pull request closed without merging**